### DISCUSSION TITLE
Dependencies

### DISCUSSION THREAD

**Discussion by CodyPrince** [OrcaSlicer/OrcaSlicer#1092] [Ideas]
In Superslicer and Prusaslicer there are Dependencies. Under Print settings, Filament settings, and Print settings, you can set Dependencies. Any chance this can get pushed over to Orcaslicer, or is it already there and i just dont see it?

For example, If i have a printer with a .6 nozzle, no Print settings show that aren't setup for that size nozzle. 

Thanks, Cody

**Comment by SoftFever** [comment]
@CodyPrince 
OrcaSlicer doesn't have the UI for this at the moment.
Can you create a Feature Request for this?
